libs/solid/quiz/src/lib/components/drag-and-drop-choice-question/drag-and-drop-question.component.html

```diff
@@ -1 +1,17 @@
-<p>drag-and-drop-choice-question works!</p>
+<div cdkDropList class="example-list" (cdkDropListDropped)="drop($event)">
+    <div [ngStyle]="{'background-color': result[i]?.status }" 
+    class="example-box" *ngFor="let movie of movies; let i = index" cdkDrag>
+          {{movie}} {{result[i]?.text}}
+    </div>
+    
+  </div>
+   
+  <div *ngIf = "!isShow" class="example-list" style="padding-top: 50px;">
```

Comment:
`style="padding-top: 50px;` This style is not dynamic. Please move it to the component stylesheet.